app/views/finders/show.html.erb

```diff
@@ -58,6 +58,10 @@
   <% end %>
 </header>
 
+<% if finder.summary %>
+  <div class="keyline"><!-- --></div>
```

Comment:
I did start with an `<hr>` but settled on a div instead. Mainly because this isn't at a paragraph level so much as separating `header` from `.filtering`

I think the cleanest way would be to use a `border-bottom` declaration on `header` but I can't do this as it will overhang into the grid gutters left and right.